### PR TITLE
Feature/batch run

### PR DESCRIPTION
**Pull request by RoaldL**
This feature enables the user to sequentially run multiple modelruns from a modelrun batch file that contains a list of modelrun names.

### CODE REVIEW

File: src/smif/cli/__init__.py [hunk 1]
```diff
@@ -435,30 +435,38 @@ def execute_model_run(args):
...
-    timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y%m%dT%H%M%S')
-
     LOGGER.info("Loading resolution data")
     load_resolution_sets(args.directory)
 
-    LOGGER.info("Getting model run definition")
-    model_run_config = get_model_run_definition(args.directory, args.modelrun)
+    if args.batchfile:
+        with open(args.modelrun, 'r') as f:
+            model_runs = f.read().splitlines()
```
Comment (tomalrussell) [inline src/smif/cli/__init__.py]: Can we fail fast here with an error if any of the model runs don't exist?

File: src/smif/cli/__init__.py [hunk 2]
```diff
@@ -435,30 +435,38 @@ def execute_model_run(args):
...
+    else:
+        model_runs = [args.modelrun]    
 
-    LOGGER.info("Build model run from configuration data")
-    modelrun = build_model_run(model_run_config)
+    for model_run in model_runs:
+        print("Start model run '" + model_run)
+        timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y%m%dT%H%M%S')
```
Comment (tomalrussell) [inline src/smif/cli/__init__.py]: Should we move timestamp up to be generated once, so it is consistent across model runs in a batch?

File: src/smif/cli/__init__.py (hunk 1 above)
Comment (RoaldL) [reply src/smif/cli/__init__.py]: Done

File: src/smif/cli/__init__.py (hunk 2 above)
Comment (RoaldL) [reply src/smif/cli/__init__.py]: Done

**Pull request merged**